score-client/src/main/java/bio/overture/score/client/command/DownloadCommand.java

```diff
@@ -61,22 +61,39 @@
   /**
    * Options
    */
-  @Parameter(names = "--output-dir", description = "Path to output directory", required = true, validateValueWith = CreatableDirectoryValidator.class)
-  private File outputDir;
-  @Parameter(names = "--output-layout", description = "Layout of the output-dir. One of 'bundle' (saved according to filename under GNOS bundle id directory), 'filename' (saved according to filename in output directory), or 'id' (saved according to object id in output directory)", converter = OutputLayoutConverter.class)
-  private OutputLayout layout = OutputLayout.FILENAME;
-  @Parameter(names = "--force", description = "Force re-download (override local file)")
-  private boolean force = false;
+  // 1) Which files to download
   @Parameter(names = "--manifest", description = "Manifest id, url, or path to manifest file")
   private ManifestResource manifestResource;
   @Parameter(names = "--object-id", description = "Object id to download", validateValueWith = ObjectIdListValidator.class, variableArity = true)
   private List<String> objectId = new ArrayList<>();
+  @Parameter(names = "--analysis-id", description = "Analysis to download")
```

Comment:
add some logic to force mutual exclusivity amongs the following sets of switches:
1. --analysis-id --study-id (these are either BOTH defined or undefined)
2. --object-id
3. --manifest

if more than one is defined, an error should be thrown since they dont make sense being called together